handle-errors/Dockerfile

```
@@ -13,6 +13,7 @@ RUN adduser \
     "${USER}"
 
 WORKDIR $GOPATH/src
+COPY go.mod ./
 COPY main.go ./
```

Style nit: the new file could be added to the existing COPY layer.

```suggestion
COPY main.go go.mod ./
```